docs/send-data/collect-from-other-data-sources/azure-monitoring/collect-metrics-azure-monitor.md

```diff
@@ -34,42 +34,42 @@ In this step, you configure an HTTP source to receive logs from the Azure functi
 
 In this step, you use a Sumo-provided Azure Resource Manager (ARM) template to create an Event Hub, an Azure function and two Storage Accounts. The Azure function is triggered by Event Hub. Two storage accounts are used to store log messages from the Azure function and failover data from Event Hub. 
 
-1. Download the [azuredeploy_metrics.json](https://s3.amazonaws.com/appdev-cloudformation-templates/azuredeploy_metrics.json) ARM template.
+1. Download the [azuredeploy_metrics.json](https://raw.githubusercontent.com/SumoLogic/sumologic-azure-function/master/EventHubs/src/azuredeploy_metrics.json) ARM template.
 1. Go to **Template deployment** in the Azure Portal.
+    **Template deployment (deploy using custom templates)** under **Marketplace**
+    ![azure-template-deploy.png](/img/send-data/azure-metrics/azure-template-deploy.png)
 
-    ![azure-template-deploy.png](/img/send-data/azure-template-deploy.png)
-
-1. Click **Create**.
 1. On the **Custom deployment** blade, click **Build your own template in the editor.**
+    ![custom-deployemnt.png](/img/send-data/azure-metrics/custom-deployemnt.png)
+
 1. Copy the contents of `azuredeploy_metrics.json` and paste it into the editor window, and click **Save**.
 
-    ![azure-template-deploy.png](/img/send-data/azure-template-deploy2.png)
+    ![azure-template-deploy.png](/img/send-data/azure-metrics/azure-template-deploy2.png)
 
 1. Now you are back on the **Custom deployment** blade.
 
    1. Create a new Resource Group (recommended) or select an existing one.
-   1. Choose **Location**.
+   1. Choose **Region**.
```

Comment:
Choose **Region**. The event hub namespace needs to be in the same region as the resource being monitored if the resource is regional.

Reply:
Updated